public/content/documentation/web/page-level/header-landmark.md

```diff
@@ -1,69 +1,56 @@
+   - Tab: Nothing, headers are not focusable unless they are actionable
+   - Arrow-keys: headers are browsed
 
 2. Test mobile screenreader gestures
-
-   &mdash; Swipe: Focus moves directly to the header or navigation
-
-   &mdash; Doubletap: This typically activates most elements
+   - Swipe: Focus moves directly to the header or navigation
+   - Doubletap: This typically activates most elements
 
 3. Listen to screenreader output on all devices
-
-   &mdash; Role: It is discoverable with screenreader shortcuts as header/banner landmark
-
-   &mdash; Group: It typically contains the name and primary navigation of the website
+   - Role: It is discoverable with screenreader shortcuts as header/banner landmark
+   - Group: It typically contains the name and primary navigation of the website
 
 Full information: https://www.magentaa11y.com/#/web-criteria/page-level/header-landmark
 
 ## Gherkin
 
-### a11y - Web Accessibility Acceptance Criteria
+### #a11y - Web Accessibility Acceptance Criteria
 
 How to test a header landmark
 
 GIVEN THAT I am on a page with a header landmark
 
 1. Keyboard for mobile & desktop
-
-   &mdash; WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
 
 2. Desktop screenreader
-
-   &mdash; WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
-
-   &mdash; I use the tab key to enter the web browser window
-
-   &mdash; I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
-
-   &mdash; I HEAR It typically contains the name and primary navigation of the website
-
+    - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+    - I use the tab key to enter the web browser window
+       - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+       - I HEAR It typically contains the name and primary navigation of the website
 3. Mobile screenreader
-
-   &mdash; WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
-
-   &mdash; I swipe to focusable elements in the header
-
-   &mdash; I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
-
-   &mdash; I HEAR It typically contains the name and primary navigation of the website
+    - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+    - I swipe to focusable elements in the header
+       - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+       - I HEAR It typically contains the name and primary navigation of the website
 
 Full information: https://www.magentaa11y.com/#/web-criteria/page-level/header-landmark
 
+## General
+* There must only be a singular header/banner element on the page. 
+* Contains the site title and typically the primary navigation.
+
 ## Code Examples
 
 ### Use Semantic HTML
```

Comment:
Between the code sample and "When you can't use semantic HTML", current MagentaA11Y includes the mock up of the code sample which is not featured on the new MagentaA11y.

Reply:
yeah I was trying to decided if that current day demo really added anything. What do you think? Should I include it?
<img width="464" alt="Screenshot 2025-04-21 at 8 31 43 PM" src="https://github.com/user-attachments/assets/390cb49d-527d-4002-bd0d-812f406cc4ca" />

Reply:
We can always go back and include them later if that's the way discussion goes, that's alright!